hack/local-up-kubeedge.sh

```diff
@@ -141,7 +141,7 @@ build_edgecore
 
 kind_up_cluster
 
-export KUBECONFIG="$(kind get kubeconfig-path ${CLUSTER_CONTEXT})"
+export KUBECONFIG=$HOME/.kube/config
```

Comment:
This is the fix
```suggestion
export KUBECONFIG="$(kind ${CLUSTER_CONTEXT}) get kubeconfig-path"
```

Reply:
```shell
➜  ~ kind get
Gets one of [clusters, nodes, kubeconfig]

Usage:
  kind get [command]

Available Commands:
  clusters    Lists existing kind clusters by their name
  kubeconfig  Prints cluster kubeconfig
  nodes       Lists existing kind nodes by their name

Flags:
  -h, --help   help for get

Global Flags:
      --loglevel string   DEPRECATED: see -v instead
  -q, --quiet             silence all stderr output
  -v, --verbosity int32   info log verbosity

Use "kind get [command] --help" for more information about a command.
```
Looks like the  `kubeconfig-path` subcomd  has been removed from `kind get`?

Reply:
from kind v0.6.0 release note:
You do not need to and should not do export KUBECONFIG="$(kind get kubeconfig-path)". This command is deprecated and will be removed in a future release.
kind will set your current-context when creating a cluster
You can configure using a different file by either setting the KUBECONFIG environment variable before creating a cluster, or by setting the --kubeconfig flag on cluster creation
https://github.com/kubernetes-sigs/kind/releases/tag/v0.6.0